manifest/docker_schema2_list_test.go

```diff
@@ -1,28 +0,0 @@
-package manifest
```

Comment:
Let’s preserve this, to have _something_ to exercise the public interface. The more we move our own callers to the internal-only API, the more useful this is, even if it is a small smoke test.

Same for `oci_index_test.go`.